README.md

```diff
@@ -19,7 +27,7 @@ make
 ```
 
 ## Contributing
-Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change. Our feature list is located on [Trello](https://trello.com/b/jXkRj0IH/loris-payload)
```

Comment:
I might be worth changing the major changes line to opening a trello task. Up for debate though